python/tank/authentication/session_cache.py

```diff
@@ -432,8 +432,7 @@ def set_current_host(host):
 
     :param host: The new current host.
     """
-    if host:
-        host = connection.sanitize_url(host)
+    host = connection.sanitize_url(host)
```

Comment:
The code tried to get around the fact that sanitize might prefix empty urls with `https://`. This doesn't happen anymore, which simplifies the code.